layouts/shortcodes/terminal.html

```diff
@@ -1 +1 @@
-<i class='fas fa-terminal fa-fw'></i>
\ No newline at end of file
+$
```

Comment:
If you use font awesome's `fa-dollar-sign` instead, then it prevents users from accidentally copying the `$` if they copy/paste from the page.  For that reason, I'd prefer the `fa-dollar-sign` unless it looks really wonky.

Reply:
Ah yeah good point about the copying and pasting. I used `$` because on my terminal the symbol is exactly `$`, where the font-awesome sign is a bit different, but not different enough to warrant the copy and paste confusion.